x-pack/plugins/security_solution/public/common/components/events_viewer/event_details_flyout.tsx

```diff
@@ -40,6 +43,26 @@ const EventDetailsFlyoutComponent: React.FC<EventDetailsFlyoutProps> = ({
     (state) => state.timeline.timelineById[timelineId]?.expandedEvent ?? emptyExpandedEvent
   );
 
+  const [loading, detailsData] = useTimelineEventsDetails({
+    docValueFields,
+    indexName: expandedEvent.indexName!,
+    eventId: expandedEvent.eventId!,
+    skip: !expandedEvent.eventId,
+  });
+
+  const ruleId = useMemo(() => {
```

Comment:
maybe we could return boolean `isAlert` from that, so we don't need to do `ruleId != null` later?